packages/jni-swig-stub/realm.i

```diff
@@ -105,7 +105,7 @@ void throw_as_java_exception(JNIEnv *jenv) {
 
         // Invoke CoreErrorUtils.coreErrorAsThrowable() to retrieve an exception instance that
         // maps to the core error.
-        static jclass error_type_class = (jenv)->FindClass("io/realm/interop/CoreErrorUtils");
+        jclass error_type_class = (jenv)->FindClass("io/realm/interop/CoreErrorUtils");
```

Comment:
It does not work with Kotlin objects (final classes in java). Klint complained about the class could be an object, but you are right it would be better to ignore the klint exception and make it static.

Reply:
It seems we cannot cache classes for calling static methods. When defining it as static (companion object) it crashes with: `JNI ERROR (app bug): attempt to use stale Local 0x15 (should be 0x19)`.